agent/xdsv2/cluster_resources.go

```diff
@@ -234,6 +232,9 @@ func (pr *ProxyResources) makeEnvoyAggregateCluster(name string, protocol string
 				},
 			},
 		}
+		if fg.Config.UseAltStatName {
```

Comment:
This was already present on the standard cluster path, I just cloned that behavior to the aggregate one.